ci-operator/step-registry/openshift-qe/installer/aws/etcd-encryption/OWNERS

```diff
@@ -0,0 +1 @@
+../../OWNERS
```

Comment:
think we need 2 more ../ to reference back to the OWNERS file in openshift-qe base folder.

Reply:
I checked the path locally and it seems to point correctly to the OWNERs?